### PR TITLE
file_syncer: act like rsync and don't delete by default

### PR DESCRIPTION
**Pull request by philips**
if you want the deletion behaviour throw --delete on it

### CODE REVIEW

**Comment by Kami** [comment]
Can you please also update the readme? Otherwise, LGTM.

**Comment by philips** [comment]
@Kami readme updated.

File: file_syncer/run.py [hunk 1]
```diff
@@ -61,7 +61,9 @@ def run():
                            'exclude')
     parser.add_option('--log-level', dest='log_level', default='INFO',
                       help='Log level')
-
+    parser.add_option('--delete', dest='delete',  default=False,
```
Comment (Kami) [inline file_syncer/run.py]: I think this one is missing `action='store_true`.

Comment (philips) [inline file_syncer/run.py]: fixed in 74c17c7

File: file_syncer/syncer.py [hunk 2]
```diff
@@ -141,9 +141,10 @@ def sync(self):
             # 1 - Upload new or changed files and remove deleted ones
             # 2 - Upload manifest
 
-            for item in actions['to_remove']:
-                func = lambda item: self._remove_object(item=item, pool=pool)
-                pool.spawn(func, item)
+            if delete:
```
Comment (Kami) [inline file_syncer/syncer.py:144]: I think this is redundant here because you already do this check in `_calculate_actions` :)

Comment (Kami) [inline file_syncer/syncer.py:144]: I'll fix this myself. Merged into master, thanks!

**Comment by Kami** [comment]
New version (0.3.0) has been published to PyPi - http://pypi.python.org/pypi/file_syncer/0.3.0

**Pull request merged**